planb/common.py

```diff
@@ -138,13 +138,12 @@ def create_instance_profile(cluster_name: str):
     profile_name = make_instance_profile_name(cluster_name)
     role_name = 'role-{}'.format(cluster_name)
     policy_name = 'policy-{}-datavolume'.format(cluster_name)
-    name_tag_pattern = '{}-*'.format(cluster_name)
 
     iam = boto3.client('iam')
 
     profile = iam.create_instance_profile(InstanceProfileName=profile_name)
 
-    role = iam.create_role(RoleName=role_name, AssumeRolePolicyDocument="""{
+    iam.create_role(RoleName=role_name, AssumeRolePolicyDocument="""{
```

Comment:
This is still against the recommendation to break line before the first argument.  Should we fix this as well?